spec/truffle/truffle.mspec

```diff
@@ -28,6 +28,7 @@ class MSpecScript
     core_path = "#{JRUBY_DIR}/truffle/src/main/ruby"
     if File.directory?(core_path)
       flags << "-Xcore.load_path=#{core_path}"
+      flags << "-J-Dtruffleruby.core.build_parser_cache=false"
```

Comment:
Because we want to start building the cache as early as possible, before the command line is fully parsed.